### PR TITLE
submitting group 99 bottles hw

### PR DESCRIPTION
**Pull request by daphne4375**

### CODE REVIEW

File: Get_a_name/99_bottles/99_bottles.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+for ((i=100
+	..; i>=1; i--));do
```
Comment (nsuitter) [inline Get_a_name/99_bottles/99_bottles.sh]: @daphne4375 I'm getting a syntax error at this line. Did you mean to include the ..?

Comment (daphne4375) [inline Get_a_name/99_bottles/99_bottles.sh]: @nsuitter Sorry Nathan! It should be for ((i=99; i>=1; i--));do.  Should we repush our assignment?

Comment (nsuitter) [inline Get_a_name/99_bottles/99_bottles.sh]: Please do:)

**Comment by daphne4375** [comment]
Fixed syntax error

**Comment by nsuitter** [comment]
:+1:

**Pull request merged**